jekyll/_config_production.yml

```diff
@@ -4,3 +4,40 @@ rollbar_env: production
 
 # Optimizely SDK key for prod env
 optimizely: "2DuqgdSgVTZ1n4upLM2Her"
+
+defaults:
+  - scope:
+      path: ""
+      type: "cci2"
+    values:
+      lang: "en"
+
+  - scope:
+      path: ""
+      type: "cci2_ja"
+    values:
+      lang: "ja"
+
+  - scope:
+      path: ""
+    values:
+      hide: false
+      search: true
+
+  # Enable TOC by default
+  - scope:
+      path: ""
+    values:
+      toc: true
+
+  # Disable helpful resources section by default
+  - scope:
+      path: ""
+    values:
+      suggested: false
```

Comment:
This was the reason for the incident. Originally it was set as: 

```
defaults:
  - scope:
        path: ""
      values:
        sitemap: true
```

But that has the side effect of replacing the entire `default` object instead of merging that field with what is coming from `_config.yml`